tests/album-create.test.js

```diff
@@ -0,0 +1,48 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+
+describe('create album', () => {
+    let db;
+    let albums; 
+ 
+    beforeEach(async () => {
+        db = await getDb();
+ 
+       [albums] = await db.query('SELECT * from Album');
+    });
+ 
+    afterEach(async () => {
+      await db.query('DELETE FROM Album');
+      await db.close();
+    });
+ 
+    describe('/album', () => {
+      describe('POST', () => {
+        it('creates a new album in the database', async () => {
+ 
+         if(albums[0]) {
+          const res = await request(app).post('/artist/1/album').send({
+            name: 'Red',
+            genre: 'Country',
+            year: 2000,
+          });
+ 
+          expect(res.status).to.equal(201);
+ 
+          const [[albumEntries]] = await db.query(
+            `SELECT * FROM Album WHERE name = 'Red'`
+          );
+ 
+          expect(albumEntries.name).to.equal('Red');
+          expect(albumEntries.genre).to.equal('Country');
+          expect(albumEntries.year).to.equal(2000);
+          expect(albumEntries.artistId).to.equal(1);
+ 
+         }
+        });
+      });
+    });
+  });
```

Comment:
Normally I wouldnt write a suggestion for an entire code block like this, but thought it would be good just to serve as an example of what it should look like as a reference. Main points here are the different parts of the tests are segregated, the assertions have a blank line before them, and indentation is uniform (the functionality of the code here is fine, this is just about the layout):

```
describe('create album', () => {
  let db;
  let albums;

  beforeEach(async () => {
    db = await getDb();

    [albums] = await db.query('SELECT * from Album');
  });

  afterEach(async () => {
    await db.query('DELETE FROM Album');
    await db.close();
  });

  describe('/album', () => {
    describe('POST', () => {
      it('creates a new album in the database', async () => {
        if (albums[0]) {
          const res = await request(app).post('/artist/1/album').send({
            name: 'Red',
            genre: 'Country',
            year: 2000,
          });

          expect(res.status).to.equal(201);

          const [[albumEntries]] = await db.query(
            `SELECT * FROM Album WHERE name = 'Red'`
          );

          expect(albumEntries.name).to.equal('Red');
          expect(albumEntries.genre).to.equal('Country');
          expect(albumEntries.year).to.equal(2000);
          expect(albumEntries.artistId).to.equal(1);
        }
      });
    });
  });
});
```